lib/email_category_badge.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module EmailCategoryBadge
```

Comment:
This module is no longer being called since the logic was moved into `CategoryBadge` using the `inline_styles` option.